test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -129,6 +129,12 @@ def test_successful_authorization_with_elo
     assert !response.authorization.blank?
   end
 
+  def test_successful_authorization_with_installment_total_count
+    assert response = @gateway.authorize(@amount, @credit_card, installment_total_count: 5)
+    assert_successful_response(response)
```

Comment:
⁉️ 😐 why are there two ways to do the same thing tho

Reply:
It turns out `assert_succesful_response` is a CyberSource-test-specific helper that itself executes `assert_success response` as well as checking for a successful transaction message and that we get a test response back. Implementing it throughout the test reduced duplication pretty substantially, though I don't know of a way to ensure that future CyberSource committers use it instead of just going with what they know. 

For now I've standardized on the helper (it seems a shame not to use it) but I'm open to conversation on that.

Reply:
Woah! Reducing duplication is always awesome!

Maybe worth adding a comment somewhere, perhaps to the first instance of its use in this file, to encourage future folks to follow the pattern?

Reply:
I added a comment, and fully obliterated any examples of `assert_success response` in this file (except the one in the helper). Remote tests still generate the same failures, so ... seems okay :D

Reply:
🎉 That sounds very positive! Super exciting! 🥇